src/utils/css.js

```diff
@@ -48,6 +48,12 @@ function classNames(prefix, modifierArgs) {
     return classes.join(' ')
 }
 
+export function componentStyles(styles) {
```

Comment:
We can export all styles into a single JSON and use it here, but that would mean that even if a user only wants a **Button** s/he would have to import the entire JSON file which maps classes to CSS modules.